README.md

```diff
@@ -1 +1,28 @@
-# wor-paginate
\ No newline at end of file
+# Wor::Paginate
+Short description and motivation.
+
+## Usage
+How to use my plugin.
+
+## Installation
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'wor-paginate'
+```
+
+And then execute:
+```bash
+$ bundle
+```
+
+Or install it yourself as:
+```bash
+$ gem install wor-paginate
+```
+
+## Contributing
```

Comment:
Copy from contributing till the end https://github.com/Wolox/wor-authentication/blob/master/README.md#contributing